### DISCUSSION TITLE
.env file doesn't work

### DISCUSSION THREAD

**Discussion by HiddenUndefined** [nuxt/nuxt#18328] [Questions] [answered]
When I tried user `.env` variables (example: `process.env.APP_GOOGLE_MAP_TOKEN`), I get `undefined`.

I created `.env` file in project root directory and created `APP_GOOGLE_MAP_TOKEN="****"` vairable in my `.env` file, but can't get my variable value in component `script`.

Where am I wrong?

**Comment by wobsoriano** [comment, marked as answer]
Related to this https://github.com/nuxt/framework/discussions/1000
